backend/model/morphware.js

```diff
@@ -1,178 +1,190 @@
-'use strict';
```

Comment:
This class manages the actual instance of the wallet that the client will act on behalf of. So on startup, it will use the PK provided to instantiate an instance of the wallet. It also provides wrapper functions to approve/send/add balance/ get transaction history/ MWT price and a few others.

Anything related to the wallet and MWT price is managed in this class

Reply:
Got it thanks! What's PK?

Reply:
PK is the Private Key of the wallet